packages/dev-server/src/createKoaServer.ts

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs-extra';
+import Koa from 'koa';
```

Comment:
IMO we should opt for fastify: it's faster and has a better set of plugins that we could use